nordicNRPlus/processNrplusMessagesAndUpdateThingShadow.ts

```diff
@@ -0,0 +1,41 @@
+import type { Static } from '@sinclair/typebox'
+import type { inputSchemaLwm2mMessage } from '../lambda/webhookHandler.ts'
```

Comment:
Move this here, so you do not include the lambda on unit tests, because lambdas access global state and do other stuff (e.g.: 

```typescript
export const iotData = new IoTDataPlaneClient({})
const iotClient = new IoTClient({})
```